### DISCUSSION TITLE
Human readable time since last upload

### DISCUSSION THREAD

**Discussion by expipiplus1** [flora-pm/flora-server#382] [Feature Requests]
One of the bits of information I'm usually looking for on a hackage or flora page is the date of upload, specifically to see if a package has been languishing for a decade, or is likely to work with a recent GHC.

Would it make sense for Flora to display this directly (i.e. the time since the most recent publishing rather than just the date) and maybe also a little more prominently?

See also: https://github.com/haskell/hackage-server/issues/873

**Comment by tchoutri** [comment]
That should be easily doable with DayJS, at the very least.

**Reply by tchoutri** [reply]
@expipiplus1 What does your intuition tell you about the potential location of this relative time indication? 

![Screenshot 2023-05-19 at 13-17-04 @haskell_text](https://github.com/flora-pm/flora-server/assets/15848443/b7651fa9-588f-4f57-acef-b0bdcfe04afb)

**Reply by expipiplus1** [reply]
Anywhere, as long as it's consistent. One of the problems with Hackage's presentation is that it's in amongst other variable package info which makes it hard to find without a linear search with the eyes.

Semantically, next to the versions maybe?

**Reply by tchoutri** [reply]
Alright let's start with this
